content/en/platform/corda/4.10/community/api-stability-guarantees.md

```diff
@@ -20,18 +20,16 @@ title: API stability guarantees
 
 # API stability guarantees
 
-Corda makes certain commitments about what parts of the API will preserve backwards compatibility as they change and
+Corda makes certain commitments about what parts of the Corda 4 API will preserve stability as they change and
```

Comment:
Now I'm not sure if you want the "as they change" phrase in or not? I think the 4.8 file reads cleaner without it.  

```suggestion
Corda makes certain commitments about what parts of the Corda 4 API will preserve stability and
```